docs/src/userguide/numpy_tutorial.rst

```diff
@@ -143,6 +146,62 @@ in your Cython code.
 This creates :file:`yourmod.so` in the same directory, which is importable by
 Python by using a normal ``import yourmod`` statement.
 
+
+Compilation using setuptools
+----------------------------
+
+Setuptools allows us to create setup.py file to automate both compilation of Cython files and generated C files.::
+
+    from setuptools import Extension, setup
+    from Cython.Build import cythonize
+    import numpy
+
+    extensions = [
+        Extension("*", ["*.pyx"],
+            library_dirs=[numpy.get_include()]),
```

Comment:
`library_dirs`?

Reply:
Yes, I forgot to push the commit fixing it :-/